plugin/trace/README.md

```diff
@@ -47,6 +47,8 @@ You can run Zipkin on a Docker host like this:
 docker run -d -p 9411:9411 openzipkin/zipkin
 ```
 
+:warning: The zipkin provider does not support the v1 API since coredns 1.7.1
```

Comment:
is this useful information for the reader or can we just not mention it?

Reply:
This is a "breaking"  change compared to pre-1.7.1
I don't mind removing the note I made it in response to this [comment](https://github.com/coredns/coredns/pull/4171#issuecomment-703038038)